tests/Microsoft.Data.Analysis.IO.Tests/Microsoft.Data.Analysis.IO.Tests/Microsoft.Data.Analysis.IO.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
This needs to target both net framework and netcore

Reply:
Am I missing something, but I see just a target for .NET 5.
If this is intentional, please ignore my comment.

Reply:
Bad comment on my part. Good catch! Resolved :)